dev-utils/build.js

```diff
@@ -208,7 +208,12 @@ function getCommonWebpackConfig(bundleType, packageType) {
         }
       ]
     },
-    plugins: [new EnvironmentPlugin(getWebpackEnv(mode))],
+    plugins: [
+      new EnvironmentPlugin(getWebpackEnv(mode)),
+      new ProvidePlugin({
```

Comment:
Webpack removed polyfilling node by default from 5 so we polyfill it where required. - https://github.com/webpack/webpack/pull/8460/commits/a68426e9255edcce7822480b78416837617ab065